mobile/src/employee-details/employee-details-events-list.tsx

```diff
@@ -29,8 +32,39 @@ interface EmployeeDetailsEventsListDispatchProps {
 
 //============================================================================
 class EmployeeDetailsEventsListImpl extends Component<EmployeeDetailsEventsListProps & EmployeeDetailsEventsListDispatchProps> {
+
     private readonly eventDigitsDateFormat = 'ddd DD/MM/YYYY';
 
+    private readonly vacationDescriptions: CalendarEventStatusDescriptions = {
+        [CalendarEventStatus.Requested]: 'waiting for approvals',
```

Comment:
is it possible to split CalendarEventStatus on three different types? Changes are coming from server side when you'll have to change this mapping

Reply:
What kind of server changes do you mean?

Reply:
#694 #679 
Even bigger difference in statuses set.

Reply:
It would be great to just have a table with all statuses and their flows for particular events types instead of digging other PRs. -)